tests/jupyterhub_api/__init__.py

```diff
@@ -0,0 +1,2 @@
+import os, sys; sys.path.append(os.path.dirname(os.path.realpath(__file__)))
+sys.path.append('..')
```

Comment:
why all of this? Try to be more explicit. I don't know what is being imported nor if it's necessary.

Reply:
@diocas I have added this to avoid relative import errors. Ref -https://stackoverflow.com/questions/16981921/relative-imports-in-python-3/28151907#answer-49375740